packages/ketcher-react/src/script/ui/component/form/input.jsx

```diff
@@ -26,12 +26,15 @@ export function GenericInput({
   onChange,
   type = 'text',
   isFocused = false,
+  searchFocusBack = false,
   ...props
 }) {
   const inputRef = useRef(null)
   useEffect(() => {
-    if (inputRef.current && isFocused) inputRef.current.focus()
-  }, [inputRef, isFocused])
+    if (inputRef.current && isFocused) {
```

Comment:
Can we do the same logic in tabs, not in GenericInput? It belongs to tabs changing, but not for input itself.
I see the solution the following way in tabs component:
```
const inputRef = useRef(null)
...
const onTabChange = () => {
  ... 
  inputRef.current.focus();
}
...
return (
 ...
 <GenericInput ref={inputRef} />
 ...
)
```

Reply:
Fixed